packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -1018,40 +1027,45 @@ class OAuthController {
                 };
             }
 
-            if (connectionConfig['oauth_client_id'] && connectionConfig['oauth_client_secret']) {
+            if (connectionConfig['oauth_client_id_override']) {
```

Comment:
it would great to type connectionConfig and BaseConnection["connection_config"] in the future

Reply:
Yeah, might be hard to nail down but would be better for sure.